articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -41,8 +39,6 @@ In this tutorial, you learn how to:
 
 ## Prerequisites
 
-To do this tutorial, install the [.NET Core SDK](https://dotnet.microsoft.com/download).
-
```

Comment:
Why is everything deleted?

Reply:
I thought you wanted these lines removed:
![image](https://user-images.githubusercontent.com/89044647/151019184-fd6675e3-93b6-440b-952b-67e5baad93d1.png)

Reply:
or is this just referencing the quickstarts-free-trial-note?

Reply:
> or is this just referencing the quickstarts-free-trial-note?

yes